remotespark/MessagePrinter.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2015  aggftw@gmail.com
```

Comment:
Files should only have lowercase characters. DO NOT check this in like it is. It's a pain to change it later.